web/docs/Frontend Starters/Next.js/Next.js + WordPress/setting-environment-variables.md

```diff
@@ -23,6 +23,64 @@ WPGRAPHQL_URL=https://my-wordpress-site.pantheon.site/wp/graphql
 IMAGE_DOMAIN=my-image-cdn.site
 ```
 
+## Connecting to Multidev Cloud Environments
+
+To connect to a Multidev environment, the following helper environment variable
+can be used inside of `next.config.js`.
+
+- `IS_LIVE_ENVIRONMENT` - Automatically set to true if
+  `PANTHEON_ENVIRONMENT_URL` is live and false otherwise
+
+Either the `PANTHEON_CMS_ENDPOINT` or `WPGRAPHQL_URL` will need to be set.
+
+The `PANTHEON_CMS_ENDPOINT` can be mocked for local development by defining it
+in the `.env.development.local`
+
+```
+PANTHEON_CMS_ENDPOINT=
```

Comment:
Add an example here. This is needed because the `PANTHEON_CMS_ENDPOINT` is not the same format as the `WPGRAPHQL_URL`
```suggestion
PANTHEON_CMS_ENDPOINT=dev-my-wordpress-site.pantheonsite.io
```